CONTRIBUTING.md

```diff
@@ -1,196 +1,343 @@
+   are run via `make test` (or `make focused-test` for pending changes).
+    - Unit tests are used to verify the functionality of individual functions,
+      methods, structs, and packages.
+    - Simulated tests leverage [vcsim][vcsim] to verify behavior of packages
+      which interact with vSphere.
+3. [Integration tests](tests/README.md) are used to verify the behavior of the
+   product against a live environment. These tests can be run against a VMware
+   vSphere ESXi host or a VMware vCenter Server (tests only applicable to one
+   environment or the other should automatically be skipped). All integration
+   tests are automatically against vCenter Server when a change is merged to
+   `master` or a `release/*` branch. A configurable set of integration tests are
+   run when a pull request is submitted or updated. `local-integration-test.sh`
+   can be used to run tests from a development environment.
+    - Integration tests are written using the Robot Framework and divided into
+      various Groups covering areas of product functionality and Suites within
+      those Groups.
+4. Scenario tests are used for more complex verification. These are periodically
+   run using internal VMware infrastructure.
+    - Interoperability tests are used to verify functionality against various
+      supported versions of infrastructure components.
+    - Workload tests mimic a realistic customer workload.
+
+[vcsim]:https://github.com/vmware/govmomi/tree/master/vcsim
+
+### Drone
+
+[Drone][dronesrc] is used to run compile-time checks, compile-time tests, and
+integration tests on pull requests and pushed commits.
+
+By default, a pull request builds the project and runs compile-time tests,
+compile-time checks, and the "regression" integration test group. To customize
+the tests that run on a pull request, directives can be included in the body:
+
+- To skip all testing (e.g. for a work-in-progress PR), use `[skip ci]`.
+  - This must be included in the title of the pull request, not the body.
+- To run all integration tests, use `[full ci]`.
+- To run _specific_ integration tests, use `[specific ci=$test]`. This will run
+  the specified tests, suites, or groups in addition to the default regression
+  group. Examples:
   - To run the `1-01-Docker-Info` suite: `[specific ci=1-01-Docker-Info]`
   - To run all suites under the `Group1-Docker-Commands` group: `[specific ci=Group1-Docker-Commands]`
   - To run several specific suites: `[specific ci=$test1 --suite $test2 --suite $test3]`.
 - To skip running the unit tests, use `[skip unit]`.
-- To fail fast (make normal failures fatal) during the integration testing, use `[fast fail]`.
-- To specify a specific datastore you want, use `[shared datastore=nfs-datastore]`.
+- To make normal failures fatal during integration testing, use `[fast fail]`.
+- To specify a specific datastore, use `[shared datastore=nfs-datastore]`.
 - To specify the number of parallel jobs you want, use `[parallel jobs=2]`.
```

Comment:
These directives are in the PR template in the comments for reference - this comment must be deleted or the directives within the comment are utilized.